.gitignore

```diff
@@ -52,3 +52,4 @@ tesk_application.conf
 **/venv/
 exome_germline_single_sample_v1.3/
 **/*.pyc
+/codegen_java/.bsp/sbt.json
```

Comment:
what is this?

Reply:
https://www.scala-lang.org/blog/2020/10/27/bsp-in-sbt.html

Reply:
"Build Server Protocol"